src/testUtils/mocks/entities.ts

```diff
@@ -5,6 +5,7 @@ import { merge } from 'lodash';
 import sinon, { SinonStub } from 'sinon';
 
 import { ProposalDetails, ProposalStage /*, ProposalState */ } from '~/api/entities/Proposal/types';
+import { StoDetails } from '~/api/entities/Sto/types';
```

Comment:
Import this from `~/types`

Reply:
the same question here. Should I define StoDetails in ~/types instead /entities/Sto/types? Or I have to import from /entities/Sto/types?